maintainer/guide.md

```diff
@@ -13,6 +13,7 @@ To release a new version, please follow the steps below:
     * Once the image is ready, update the version in the `manifests/charts/*/Chart.yaml`.
     * Update the `./examples/dev_your_plugin` to use the released version.
     * Run `make fmt-go`.
+    * Update the `maintainer/feature_maturity_level.yaml`
```

Comment:
Update a maturity_level file inside a plugin is easier, the file can be called "MATURITY" or "LEVEL".